### PR TITLE
Create yaml input output for subroutines in mo_drydep.F90

### PR DESCRIPTION
**Pull request by mingxuanwupnnl**
create yaml/python input/output for the following subroutines in mo_drydep.F90

calculate_uustar
calculate_ustar
calculate_ustar_over_water
calculate_obukhov_length
calculate_aerodynamic_and_quasilaminar_resistance
calculate_resistance_rgsx_and_rsmx
calculate_resistance_rclx
calculate_resistance_rlux
calculate_gas_drydep_vlc_and_flux

xx_ts_100 location 1, unstable=false
xx_ts_300 location 1, unstable=true
xx_ts_300_loc2_has_snow location 2, index_season = 4
xx_ts_1000_loc3_over_water location3, over water surface

Let me know any new input/output is needed for yaml/py files at this stage.

I'll remove lchnk and comment out YAML_GAS_DRYDEP conditional compiling flag later.

### CODE REVIEW

File: components/eam/src/chemistry/mozart/mo_drydep.F90 [hunk 1]
```diff
@@ -376,6 +378,7 @@ subroutine drydep_xactive( lchnk, ncol, latndx,  &
 
     use seq_drydep_mod, only: seq_drydep_setHCoeff
     use physconst,      only: tmelt
+    use yaml_input_file_io
```
Comment (tangshuaiqi) [inline components/eam/src/chemistry/mozart/mo_drydep.F90]: This should already be included in `common_uses.ymlf90`.

**Review by tangshuaiqi — CHANGES_REQUESTED** [review]
Just have a few comments on clean up the original code.

Okay, just saw your last line of comment. So just have one minor comment left

**Review by tangshuaiqi — APPROVED** [review]

**Review by jeff-cohere — APPROVED** [review]

**Comment by mingxuanwupnnl** [comment]
@jeff-cohere I just saw your discussion with @brigaudet for string output. Would you suggest the same for `calculate_gas_drydep_vlc_and_flux` as well? You or someone take a note about the string array, I then delete it from yaml and py files? Thank you

**Comment by jeff-cohere** [comment]
Thanks for the note, @mingxuanwupnnl . I see that you're writing out `solsym` in a similar way, with everything already quoted. I think it's okay to leave things in since they're already properly quoted (especially if these arrays change between tests). The quoted strings don't hurt anything, and they actually are informative for testing purposes. But I can go either way in this case.

**Review by singhbalwinder — APPROVED** [review]
Looks good! In the final cleanup, please address @tangshuaiqi 's comment as well.

**Comment by mingxuanwupnnl** [comment]
Final cleanup done. All tests passed.

comp_tmp_mam4_org_v2_baselines: 9 tests
  PASS ERP_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS ERS.ne4pg2_oQU480.F2010.compy_intel
  PASS PEM_P64x1.ne4pg2_oQU480.F2010.compy_intel
  PASS PET.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P1x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D_Ln5_P32x1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_D.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS_Ly1.ne4pg2_oQU480.F2010.compy_intel
  PASS SMS.ne30pg2_EC30to60E2r2.F2010.compy_intel

**Pull request merged**